Gems/RemoteTools/Code/Platform/Android/PAL_android.cmake

```diff
@@ -0,0 +1,4 @@
+
+set(PAL_TRAIT_REMOTETOOLS_SUPPORTED TRUE)
+set(PAL_TRAIT_REMOTETOOLS_TEST_SUPPORTED TRUE)
+set(PAL_TRAIT_REMOTETOOLS_EDITOR_TEST_SUPPORTED TRUE)
```

Comment:
Repeat: This should be set to FALSE for android.
It is not applicable for the Editor